common/domain/src/main/kotlin/no/nav/mulighetsrommet/domain/dto/OpprettTiltakstypeDto.kt

```diff
@@ -0,0 +1,73 @@
+package no.nav.mulighetsrommet.domain.dto
+
+import kotlinx.serialization.Serializable
+import no.nav.mulighetsrommet.domain.serializers.LocalDateSerializer
+import java.time.LocalDate
+
+@Serializable
+data class OpprettTiltakstypeDto(
```

Comment:
Laget en ny dto for å bare holde ting adskilt foreløpig. Kanskje skal disse feltene direkte inn i TiltakstypeDto istedenfor...

Reply:
Enig i å holde det adskilt inntil vi vet litt mer.

Hadde også foretrukket om vi allerede nå hadde klart å identifisere hva en del av feltene er/betyr slik at vi evt. kunne begynt med bedre navngiving fra første stund.

Et eksempel: hva betyr alle disse `harStatus*`-feltene? Er de egentlig bare boolean slik at vi ikke hadde trengt å videre `status` i navngiving av de (f.eks. harStatusUtdanning -> girRettTilUtdanning?), eller er de faktisk statuser som potensielt kunne hatt flere verdier enn J/N?

Eller: er det mulig å erstatt et sett av disse J/N-feltene med én felles property ala `girRettighetTil: Array<"Vedtak" | "Meldeplikt" | "Tiltakspenger" | "Utdanning" | ...>`?

Lufter bare noen tanker... :thinking:

Reply:
Min foreløpige forståelse er at harStatus-feltene tilsvarer enten `J|N` og kan mappes om til boolske felter med bedre navn enn `harStatus...` ja.

Det kan være en tanke å utforske om vi heller kan ha en liste med rettigheter ja.